stdlib/Random/src/misc.jl

```diff
@@ -220,6 +220,20 @@ function shuffle!(r::AbstractRNG, a::AbstractArray)
     return a
 end
 
+function shuffle!(r::AbstractRNG, a::AbstractArray{Bool})
+    old_sum = sum(v)
```

Comment:
```suggestion
    old_sum = count(v)
```
Neat PR! It seems like `count` is the semantically-proper function here?